### ISSUE TITLE
The blockchain is not progressing when trying to setup with QBFT consensus protocol

### ISSUE THREAD

**Issue by JDawg287** [Consensys/quorum-kubernetes#200]
Hello everyone,

We are trying to set up a Kubernetes cluster on AWS and want to run a private GoQuorum network with the QBFT consensus protocol on it for some local testing. We created a Kubernetes cluster as mentioned in the [documentation](https://docs.goquorum.consensys.net/tutorials/kubernetes/create-cluster). After that, we tried to install the Helm charts with the provided values without modification. The nodes are connecting to each other and can be observed on Grafana as well. The problem is that the blockchain would not start and no new blocks are being created.

![image](https://github.com/Consensys/quorum-kubernetes/assets/38837406/ba332c56-35e5-46ee-9d8b-c6c4000f6567)

The logs from `validator1` shows the following:
```
INFO [07-06|14:31:23.542] Running with private transaction manager disabled - quorum private transactions will not be supported 
INFO [07-06|14:31:23.544] Maximum peer count                       ETH=50 LES=0 total=50
INFO [07-06|14:31:23.544] Smartcard socket not found, disabling    err="stat /run/pcscd/pcscd.comm: no such file or directory"
WARN [07-06|14:31:23.554] Found deprecated node list file /data/quorum/geth/static-nodes.json, please use the TOML config file instead. 
ERROR[07-06|14:31:23.557] Node URL enode://4417debded745dad5a9acfb7c27a8724ace9f864fcdc3e8273f712bde7e29bcbc9e33a43bceb4250e30e5a053fc915256f73dece03232bdfa1c41c3f0e0dce44@goquorum-node-validator-4-0.goquorum-node-validator-4.quorum.svc.cluster.local:30303?discport=0: lookup goquorum-node-validator-4-0.goquorum-node-validator-4.quorum.svc.cluster.local: no such host
 
INFO [07-06|14:31:23.558] Enabling recording of key preimages since archive mode is used 
INFO [07-06|14:31:23.558] Set global gas cap                       cap=25,000,000
INFO [07-06|14:31:23.558] Allocated cache and file handles         database=/data/quorum/geth/chaindata cache=16.00MiB handles=16
INFO [07-06|14:31:23.585] Persisted trie from memory database      nodes=5 size=779.00B time="24.255µs" gcnodes=0 gcsize=0.00B gctime=0s livenodes=1 livesize=0.00B
Fatal: Failed to write genesis block: database contains incompatible genesis (have 81439a26b4b1a99c46d31ede624057bd027e1327b29a0d111490143c05fed0a9, new 0f9b00eb64698fccb4de4aa92cb44ed016cbe3e68a0ac9664d9654c41802e7a0)
Fatal: Failed to write genesis block: database contains incompatible genesis (have 81439a26b4b1a99c46d31ede624057bd027e1327b29a0d111490143c05fed0a9, new 0f9b00eb64698fccb4de4aa92cb44ed016cbe3e68a0ac9664d9654c41802e7a0)
INFO [07-06|14:31:23.634] Starting pprof server                    addr=http://0.0.0.0:9545/debug/pprof
INFO [07-06|14:31:23.634] Enabling metrics collection 
INFO [07-06|14:31:23.634] Running with private transaction manager disabled - quorum private transactions will not be supported 
INFO [07-06|14:31:23.636] Maximum peer count                       ETH=50 LES=0 total=50
INFO [07-06|14:31:23.636] Smartcard socket not found, disabling    err="stat /run/pcscd/pcscd.comm: no such file or directory"
WARN [07-06|14:31:23.636] Option nousb is deprecated and USB is deactivated by default. Use --usb to enable 
DEBUG[07-06|14:31:23.636] FS scan times                            list="30.613µs" set="2.279µs" diff="1.539µs"
TRACE[07-06|14:31:23.636] Started watching keystore folder         path=/data/quorum/keystore
TRACE[07-06|14:31:23.636] Handled keystore changes                 time="84.94µs"
WARN [07-06|14:31:23.636] Found deprecated node list file /data/quorum/geth/static-nodes.json, please use the TOML config file instead. 
ERROR[07-06|14:31:23.638] Node URL enode://4417debded745dad5a9acfb7c27a8724ace9f864fcdc3e8273f712bde7e29bcbc9e33a43bceb4250e30e5a053fc915256f73dece03232bdfa1c41c3f0e0dce44@goquorum-node-validator-4-0.goquorum-node-validator-4.quorum.svc.cluster.local:30303?discport=0: lookup goquorum-node-validator-4-0.goquorum-node-validator-4.quorum.svc.cluster.local: no such host
 
DEBUG[07-06|14:31:23.639] Sanitizing Go's GC trigger               percent=100
WARN [07-06|14:31:23.639] WARNING: The flag --istanbul.blockperiod is deprecated and will be removed in the future, please use ibft.blockperiodseconds on genesis file 
INFO [07-06|14:31:23.639] Enabling recording of key preimages since archive mode is used 
INFO [07-06|14:31:23.639] Set global gas cap                       cap=25,000,000
INFO [07-06|14:31:23.639] Running with private transaction manager disabled - quorum private transactions will not be supported 
WARN [07-06|14:31:23.639] Sanitizing invalid miner gas price       provided=0 updated=0
INFO [07-06|14:31:23.639] Allocated trie memory caches             clean=154.00MiB dirty=256.00MiB
INFO [07-06|14:31:23.639] Allocated cache and file handles         database=/data/quorum/geth/chaindata cache=768.00MiB handles=524,288
DEBUG[07-06|14:31:23.666] Chain freezer table opened               database=/data/quorum/geth/chaindata/ancient table=diffs items=0 size=0.00B
DEBUG[07-06|14:31:23.667] Chain freezer table opened               database=/data/quorum/geth/chaindata/ancient table=headers items=0 size=0.00B
DEBUG[07-06|14:31:23.668] Chain freezer table opened               database=/data/quorum/geth/chaindata/ancient table=hashes  items=0 size=0.00B
DEBUG[07-06|14:31:23.670] Chain freezer table opened               database=/data/quorum/geth/chaindata/ancient table=bodies  items=0 size=0.00B
DEBUG[07-06|14:31:23.670] Chain freezer table opened               database=/data/quorum/geth/chaindata/ancient table=receipts items=0 size=0.00B
INFO [07-06|14:31:23.670] Opened ancient database                  database=/data/quorum/geth/chaindata/ancient readonly=false
DEBUG[07-06|14:31:23.671] Current full block not old enough        number=0 hash=81439a..fed0a9 delay=3,162,240
DEBUG[07-06|14:31:23.671] Account Extra Data root                  hash=000000..000000
INFO [07-06|14:31:23.671] Initialised chain configuration          config="{ChainID: 1337 Homestead: 0 DAO: <nil> DAOSupport: false EIP150: 0 EIP155: 0 EIP158: 0 Byzantium: 0 IsQuorum: true Constantinople: 0 TransactionSizeLimit: 64 MaxCodeSize: 0 Petersburg: 0 Istanbul: 0, Muir Glacier: 0, Berlin: 0  Catalyst: <nil> YOLO v3: <nil> PrivacyEnhancements: <nil> PrivacyPrecompile: <nil> EnableGasPriceBlock: <nil> Engine: unknown}"
INFO [07-06|14:31:23.677] Initialising Ethereum protocol           network=10 dbversion=8
INFO [07-06|14:31:23.677] Initialising Quorum consensus protocol   name=istanbul versions="[100 99 64]" network=10 dbversion=8
INFO [07-06|14:31:23.680] Using SegregatedCacheProvider. 
DEBUG[07-06|14:31:23.680] Account Extra Data root                  hash=000000..000000
INFO [07-06|14:31:23.680] Loaded most recent local header          number=0 hash=81439a..fed0a9 td=1 age=6y3mo3w
INFO [07-06|14:31:23.680] Loaded most recent local full block      number=0 hash=81439a..fed0a9 td=1 age=6y3mo3w
INFO [07-06|14:31:23.680] Loaded most recent local fast block      number=0 hash=81439a..fed0a9 td=1 age=6y3mo3w
DEBUG[07-06|14:31:23.680] Account Extra Data root                  hash=000000..000000
DEBUG[07-06|14:31:23.680] Account Extra Data root                  hash=000000..000000
TRACE[07-06|14:31:23.680] header data not found in ancient or level db hash=000000..000000
DEBUG[07-06|14:31:23.681] Loaded snapshot journal                  diskroot=bccda6..d3a9d0 diffhead=bccda6..d3a9d0
DEBUG[07-06|14:31:23.681] Account Extra Data root                  hash=000000..000000
DEBUG[07-06|14:31:23.681] Account Extra Data root                  hash=000000..000000
DEBUG[07-06|14:31:23.681] Reinjecting stale transactions           count=0
INFO [07-06|14:31:23.681] Loaded local transaction journal         transactions=0 dropped=0
INFO [07-06|14:31:23.681] Regenerated local transaction journal    transactions=0 accounts=0
DEBUG[07-06|14:31:23.681] Recalculated downloader QoS values       rtt=20s confidence=1.000 ttl=1m0s
DEBUG[07-06|14:31:23.682] registering qouorum protocol             protoName=istanbul version=100
DEBUG[07-06|14:31:23.682] registering a legacy protocol            protoName=istanbul version=99
DEBUG[07-06|14:31:23.682] registering a legacy protocol            protoName=istanbul version=64
TRACE[07-06|14:31:23.682] BFT: loaded voting snapshot from database snap.number=0 snap.hash=0x81439a26b4b1a99c46d31ede624057bd027e1327b29a0d111490143c05fed0a9 snap.epoch=30000 snap.validators="[0x2231f2e15F262417b1AE28D90c045505525aeF2a 0x6158756df505EA7e1E7516B2A0CaAC9B3cebF194 0xd7f55692dEA53abc4884817ABDe09D12e375DDF7 0xE3d57B3fA9E8B857404c3284486B39948dfa5139]" snap.votes=[]
DEBUG[07-06|14:31:23.682] Account Extra Data root                  hash=000000..000000
DEBUG[07-06|14:31:23.682] Account Extra Data root                  hash=000000..000000
TRACE[07-06|14:31:23.682] header data not found in ancient or level db hash=000000..000000
WARN [07-06|14:31:23.682] Unclean shutdown detected                booted=2023-07-05T22:50:41+0000 age=15h40m42s
WARN [07-06|14:31:23.682] Unclean shutdown detected                booted=2023-07-05T22:50:41+0000 age=15h40m42s
WARN [07-06|14:31:23.682] Unclean shutdown detected                booted=2023-07-05T22:50:55+0000 age=15h40m28s
WARN [07-06|14:31:23.682] Unclean shutdown detected                booted=2023-07-05T22:51:22+0000 age=15h40m1s
WARN [07-06|14:31:23.682] Unclean shutdown detected                booted=2023-07-05T22:52:07+0000 age=15h39m16s
WARN [07-06|14:31:23.682] Unclean shutdown detected                booted=2023-07-05T22:53:31+0000 age=15h37m52s
WARN [07-06|14:31:23.682] permission-config.json file is missing. Smart-contract-based permission service will be disabled error="stat /data/quorum/permission-config.json: no such file or directory"
INFO [07-06|14:31:23.682] No plugins to initialise 
INFO [07-06|14:31:23.682] Starting peer-to-peer node               instance=Geth/v1.10.3-stable-fa2f8544(quorum-v22.7.1)/linux-amd64/go1.16.15
DEBUG[07-06|14:31:23.711] TCP listener up                          addr=[::]:30303
TRACE[07-06|14:31:23.711] Adding static node                       id=e8c83a843e5db149 ip=10.101.132.99 added=true
TRACE[07-06|14:31:23.711] Starting p2p dial                        id=e8c83a843e5db149 ip=10.101.132.99 flag=staticdial
TRACE[07-06|14:31:23.711] Adding static node                       id=370d042631b3ec7a ip=10.101.201.13 added=true
INFO [07-06|14:31:23.711] Mapped network port                      proto=tcp extport=30303 intport=30303 interface=ExtIP(10.101.180.91)
TRACE[07-06|14:31:23.711] Starting p2p dial                        id=370d042631b3ec7a ip=10.101.201.13 flag=staticdial
TRACE[07-06|14:31:23.711] Adding static node                       id=ffec05cb11e6b4f0 ip=10.101.180.91 added=true
INFO [07-06|14:31:23.712] New local node record                    seq=13 id=ffec05cb11e6b4f0 ip=10.101.180.91 udp=0 tcp=30303
INFO [07-06|14:31:23.712] Started P2P networking                   self="enode://fea0f78db1757be4ad46fb351f3eff4395dba4b1edbfc4869c3ad4fc85b0881d6b4fdd86c58da2aa8276a66b7ef1e25cfea5be51b6e1019852a6464cb84ee18b@10.101.180.91:30303?discport=0"
INFO [07-06|14:31:23.712] Security Plugin is not enabled 
ERROR[07-06|14:31:23.712] Unavailable modules in HTTP API list     unavailable="[db shh quorum]" available="[admin debug web3 eth txpool personal istanbul miner net]"
ERROR[07-06|14:31:23.713] Unavailable modules in HTTP API list     unavailable="[db shh quorum]" available="[admin debug web3 eth txpool personal istanbul miner net]"
DEBUG[07-06|14:31:23.713] Allowed origin(s) for WS RPC interface [*] 
INFO [07-06|14:31:23.713] Security: TLS not enabled                endpoint=0.0.0.0:8545 reason="no TLSConfigurationSource found"
INFO [07-06|14:31:23.713] HTTP server started                      endpoint=[::]:8545    prefix= cors=* vhosts=* isTls=false
INFO [07-06|14:31:23.713] Security: TLS not enabled                endpoint=0.0.0.0:8546 reason="no TLSConfigurationSource found"
INFO [07-06|14:31:23.713] WebSocket enabled                        url=ws://[::]:8546
WARN [07-06|14:31:23.713] ------------------------------------------------------------------- 
WARN [07-06|14:31:23.713] Referring to accounts by order in the keystore folder is dangerous! 
WARN [07-06|14:31:23.713] This functionality is deprecated and will be removed in the future! 
WARN [07-06|14:31:23.713] Please use explicit addresses! (can search via `geth account list`) 
WARN [07-06|14:31:23.713] ------------------------------------------------------------------- 
TRACE[07-06|14:31:23.714] Quorum permissioning                     id=370d042631b3ec7a addr=10.101.201.13:30303             conn=staticdial EnableNodePermission=false DataDir=/data/quorum Current Node ID=ffec05cb11e6b4f012942f78c4a08485cd3e745b69a6ae125c0ecb5c09133f24 Node Name=Geth/v1.10.3-stable-fa2f8544(quorum-v22.7.1)/linux-amd64/go1.16.15 Dialed Dest="enode://a7bf5822b70fe4aedbf7b4f4cdf7369e7fe00bc88ede7babca109dc58840779965732abd7c5a8175469f445b884f6618e805159c5d168c242a879422763a5384@10.101.201.13:30303?discport=0" Connection ID=370d042631b3ec7a Connection String=370d042631b3ec7a6fecde9e0455d6dc51c8de9652e36d04086a954ec82d8998
TRACE[07-06|14:31:23.714] Node Permissioning is Disabled.          id=370d042631b3ec7a addr=10.101.201.13:30303             conn=staticdial
DEBUG[07-06|14:31:23.714] Adding p2p peer                          peercount=1 id=370d042631b3ec7a conn=staticdial addr=10.101.201.13:30303             name=Geth/v1.10.3-stable-...
TRACE[07-06|14:31:23.714] Starting protocol eth/66                 id=370d042631b3ec7a conn=staticdial
TRACE[07-06|14:31:23.714] Starting protocol istanbul/100           id=370d042631b3ec7a conn=staticdial
TRACE[07-06|14:31:23.714] Starting protocol snap/1                 id=370d042631b3ec7a conn=staticdial
TRACE[07-06|14:31:23.719] Quorum permissioning                     id=e8c83a843e5db149 addr=10.101.132.99:30303             conn=staticdial EnableNodePermission=false DataDir=/data/quorum Current Node ID=ffec05cb11e6b4f012942f78c4a08485cd3e745b69a6ae125c0ecb5c09133f24 Node Name=Geth/v1.10.3-stable-fa2f8544(quorum-v22.7.1)/linux-amd64/go1.16.15 Dialed Dest="enode://0207a309658c6e8f371c047db55159f5ea5c9e00a7c97ebc58406968464f2bf5a5f421395184e8e172978d123366bf0b5c1b9f22f0bd5d6a903b5f315df57d68@10.101.132.99:30303?discport=0" Connection ID=e8c83a843e5db149 Connection String=e8c83a843e5db149e14fb651ab4ba6d84dadcdb601f085d0396cced0d4c41a87
TRACE[07-06|14:31:23.719] Node Permissioning is Disabled.          id=e8c83a843e5db149 addr=10.101.132.99:30303             conn=staticdial
DEBUG[07-06|14:31:23.719] Adding p2p peer                          peercount=2 id=e8c83a843e5db149 conn=staticdial addr=10.101.132.99:30303             name=Geth/v1.10.3-stable-...
TRACE[07-06|14:31:23.719] Starting protocol eth/66                 id=e8c83a843e5db149 conn=staticdial
TRACE[07-06|14:31:23.719] Starting protocol istanbul/100           id=e8c83a843e5db149 conn=staticdial
TRACE[07-06|14:31:23.719] Starting protocol snap/1                 id=e8c83a843e5db149 conn=staticdial
DEBUG[07-06|14:31:23.720] Ethereum peer connected                  id=e8c83a843e5db149 conn=staticdial name=Geth/v1.10.3-stable-...
TRACE[07-06|14:31:23.721] Registering sync peer                    peer=e8c83a84
WARN [07-06|14:31:23.721] full p2p peer                            id=e8c83a843e5db149e14fb651ab4ba6d84dadcdb601f085d0396cced0d4c41a87 ethPeer="Peer e8c83a843e5db149 10.101.132.99:30303"
DEBUG[07-06|14:31:23.721] consensus subprotocol retrieved eth peer from peerset id=e8c83a843e5db149 conn=staticdial ethPeer.id=e8c83a843e5db149e14fb651ab4ba6d84dadcdb601f085d0396cced0d4c41a87 ProtoName=istanbul
DEBUG[07-06|14:31:23.721] Ethereum peer connected                  id=370d042631b3ec7a conn=staticdial name=Geth/v1.10.3-stable-...
TRACE[07-06|14:31:23.721] Registering sync peer                    peer=370d0426
DEBUG[07-06|14:31:23.721] Relaxed downloader QoS values            rtt=20s confidence=0.500 ttl=1m0s
WARN [07-06|14:31:23.721] full p2p peer                            id=370d042631b3ec7a6fecde9e0455d6dc51c8de9652e36d04086a954ec82d8998 ethPeer="Peer 370d042631b3ec7a 10.101.201.13:30303"
DEBUG[07-06|14:31:23.721] consensus subprotocol retrieved eth peer from peerset id=370d042631b3ec7a conn=staticdial ethPeer.id=370d042631b3ec7a6fecde9e0455d6dc51c8de9652e36d04086a954ec82d8998 ProtoName=istanbul
INFO [07-06|14:31:24.493] Unlocked account                         address=0x900E0F6844D539E34d17941aD2e01Cc22bdE50e6
INFO [07-06|14:31:24.493] Transaction pool price threshold not updated as gasPrice is not enabled 
INFO [07-06|14:31:24.493] Transaction pool price threshold not updated as gasPrice is not enabled 
INFO [07-06|14:31:24.493] BFT: activate QBFT 
TRACE[07-06|14:31:24.493] BFT: set ProposerPolicy sorter to ValidatorSortByByteFunc 
INFO [07-06|14:31:24.493] QBFT: start                              address=0xc4A08485cd3E745B69A6Ae125c0ECb5c09133F24
INFO [07-06|14:31:24.493] QBFT: initialize new round               address=0xc4A08485cd3E745B69A6Ae125c0ECb5c09133F24 old.round=-1 old.seq=0 target.round=0 lastProposal.number=0 lastProposal.hash=81439a..fed0a9
DEBUG[07-06|14:31:24.493] QBFT: start at the initial round         address=0xc4A08485cd3E745B69A6Ae125c0ECb5c09133F24 old.round=-1 old.seq=0 target.round=0 lastProposal.number=0 lastProposal.hash=81439a..fed0a9
TRACE[07-06|14:31:24.493] BFT: loaded voting snapshot from cache   snap.number=0 snap.hash=0x81439a26b4b1a99c46d31ede624057bd027e1327b29a0d111490143c05fed0a9 snap.epoch=30000 snap.validators="[0x2231f2e15F262417b1AE28D90c045505525aeF2a 0x6158756df505EA7e1E7516B2A0CaAC9B3cebF194 0xd7f55692dEA53abc4884817ABDe09D12e375DDF7 0xE3d57B3fA9E8B857404c3284486B39948dfa5139]" snap.votes=[]
DEBUG[07-06|14:31:24.493] QBFT: lookup for pending block proposal requests address=0xc4A08485cd3E745B69A6Ae125c0ECb5c09133F24 current.round=0 current.sequence=1 state="Accept request"
INFO [07-06|14:31:24.493] QBFT: start new round                    address=0xc4A08485cd3E745B69A6Ae125c0ECb5c09133F24 old.round=-1 old.seq=0 next.round=0 next.seq=1 next.proposer=0x2231f2e15F262417b1AE28D90c045505525aeF2a next.valSet="[0x2231f2e15F262417b1AE28D90c045505525aeF2a 0x6158756df505EA7e1E7516B2A0CaAC9B3cebF194 0xE3d57B3fA9E8B857404c3284486B39948dfa5139 0xd7f55692dEA53abc4884817ABDe09D12e375DDF7]" next.size=4 next.IsProposer=false
TRACE[07-06|14:31:24.494] BFT: loaded voting snapshot from cache   snap.number=0 snap.hash=0x81439a26b4b1a99c46d31ede624057bd027e1327b29a0d111490143c05fed0a9 snap.epoch=30000 snap.validators="[0x2231f2e15F262417b1AE28D90c045505525aeF2a 0x6158756df505EA7e1E7516B2A0CaAC9B3cebF194 0xd7f55692dEA53abc4884817ABDe09D12e375DDF7 0xE3d57B3fA9E8B857404c3284486B39948dfa5139]" snap.votes=[]
DEBUG[07-06|14:31:24.494] Account Extra Data root                  hash=000000..000000
DEBUG[07-06|14:31:24.494] Account Extra Data root                  hash=000000..000000
TRACE[07-06|14:31:24.494] header data not found in ancient or level db hash=000000..000000
INFO [07-06|14:31:24.494] Commit new mining work                   number=1 sealhash=9f50bd..cd4cca uncles=0 txs=0 gas=0 fees=0 elapsed="224.394µs"
TRACE[07-06|14:31:24.494] BFT: loaded voting snapshot from cache   snap.number=0 snap.hash=0x81439a26b4b1a99c46d31ede624057bd027e1327b29a0d111490143c05fed0a9 snap.epoch=30000 snap.validators="[0x2231f2e15F262417b1AE28D90c045505525aeF2a 0x6158756df505EA7e1E7516B2A0CaAC9B3cebF194 0xd7f55692dEA53abc4884817ABDe09D12e375DDF7 0xE3d57B3fA9E8B857404c3284486B39948dfa5139]" snap.votes=[]
WARN [07-06|14:31:24.494] Block sealing failed                     err=unauthorized
DEBUG[07-06|14:31:43.682] Recalculated downloader QoS values       rtt=20s confidence=0.750 ttl=1m0s
TRACE[07-06|14:31:56.356] Accepted connection                      addr=10.101.234.166:60386
TRACE[07-06|14:31:56.357] Quorum permissioning                     id=50f6f1539111fe82 addr=10.101.234.166:60386            conn=inbound    EnableNodePermission=false DataDir=/data/quorum Current Node ID=ffec05cb11e6b4f012942f78c4a08485cd3e745b69a6ae125c0ecb5c09133f24 Node Name=Geth/v1.10.3-stable-fa2f8544(quorum-v22.7.1)/linux-amd64/go1.16.15 Dialed Dest=nil Connection ID=50f6f1539111fe82 Connection String=50f6f1539111fe82f06ef4f2a92a840b057bf34c8c9b4263d48c0d878c0896f0
TRACE[07-06|14:31:56.357] Node Permissioning is Disabled.          id=50f6f1539111fe82 addr=10.101.234.166:60386            conn=inbound
DEBUG[07-06|14:31:56.363] Adding p2p peer                          peercount=3 id=50f6f1539111fe82 conn=inbound    addr=10.101.234.166:60386            name=Geth/v1.10.3-stable-...
INFO [07-06|14:31:56.363] Looking for peers                        peercount=3 tried=2 static=3
TRACE[07-06|14:31:56.363] Starting protocol eth/66                 id=50f6f1539111fe82 conn=inbound
TRACE[07-06|14:31:56.363] Starting protocol istanbul/100           id=50f6f1539111fe82 conn=inbound
TRACE[07-06|14:31:56.363] Starting protocol snap/1                 id=50f6f1539111fe82 conn=inbound
DEBUG[07-06|14:31:56.366] Ethereum peer connected                  id=50f6f1539111fe82 conn=inbound    name=Geth/v1.10.3-stable-...
TRACE[07-06|14:31:56.366] Registering sync peer                    peer=50f6f153
DEBUG[07-06|14:31:56.366] Relaxed downloader QoS values            rtt=20s confidence=0.500 ttl=1m0s
WARN [07-06|14:31:56.366] full p2p peer                            id=50f6f1539111fe82f06ef4f2a92a840b057bf34c8c9b4263d48c0d878c0896f0 ethPeer="Peer 50f6f1539111fe82 10.101.234.166:60386"
DEBUG[07-06|14:31:56.366] consensus subprotocol retrieved eth peer from peerset id=50f6f1539111fe82 conn=inbound    ethPeer.id=50f6f1539111fe82f06ef4f2a92a840b057bf34c8c9b4263d48c0d878c0896f0 ProtoName=istanbul
DEBUG[07-06|14:32:03.682] Recalculated downloader QoS values       rtt=20s confidence=0.750 ttl=1m0s
DEBUG[07-06|14:32:23.671] Current full block not old enough        number=0 hash=81439a..fed0a9 delay=3,162,240
DEBUG[07-06|14:32:23.683] Recalculated downloader QoS values       rtt=20s confidence=0.875 ttl=1m0s
DEBUG[07-06|14:32:43.684] Recalculated downloader QoS values       rtt=20s confidence=0.938 ttl=1m0s
DEBUG[07-06|14:33:03.685] Recalculated downloader QoS values       rtt=20s confidence=0.969 ttl=1m0s
DEBUG[07-06|14:33:23.672] Current full block not old enough        number=0 hash=81439a..fed0a9 delay=3,162,240
DEBUG[07-06|14:33:23.685] Recalculated downloader QoS values       rtt=20s confidence=0.984 ttl=1m0s
DEBUG[07-06|14:33:43.686] Recalculated downloader QoS values       rtt=20s confidence=0.992 ttl=1m0s
DEBUG[07-06|14:34:03.687] Recalculated downloader QoS values       rtt=20s confidence=0.996 ttl=1m0s
```

After this, the node keeps on repeating:
```
DEBUG[07-06|14:33:23.672] Current full block not old enough        number=0 hash=81439a..fed0a9 delay=3,162,240
DEBUG[07-06|14:33:23.685] Recalculated downloader QoS values       rtt=20s confidence=0.984 ttl=1m0s
DEBUG[07-06|14:33:43.686] Recalculated downloader QoS values       rtt=20s confidence=0.992 ttl=1m0s
DEBUG[07-06|14:34:03.687] Recalculated downloader QoS values       rtt=20s confidence=0.996 ttl=1m0s
```

The genesis config from `validator1` node is as follows:
```
{
  "nonce": "0x0",
  "timestamp": "0x58ee40ba",
  "extraData": "0xf87aa00000000000000000000000000000000000000000000000000000000000000000f85494d94f52575057d0695c0639bea441f95d9496e51c94288e5e9e2b3479940d1697e3ae6f68e3d009103d94a640afee125ca83910ff2b9992f7ebcde2840bc794abc17fb17e9af964900a406d383ef22bc5ba6513c080c0",
  "gasLimit": "0x47b760",
  "gasUsed": "0x0",
  "number": "0x0",
  "difficulty": "0x1",
  "coinbase": "0x0000000000000000000000000000000000000000",
  "mixHash": "0x63746963616c2062797a616e74696e65206661756c7420746f6c6572616e6365",
  "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
  "config": {
    "chainId": 1337,
    "homesteadBlock": 0,
    "eip150Block": 0,
    "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "eip155Block": 0,
    "eip158Block": 0,
    "byzantiumBlock": 0,
    "constantinopleBlock": 0,
    "petersburgBlock": 0,
    "istanbulBlock": 0,
    "muirglacierblock": 0,
    "berlinBlock": 0,
    "londonBlock": 0,
    "isQuorum": true,
    "maxCodeSizeConfig": [
      {
        "block": 0,
        "size": 64
      }
    ],
    "txnSizeLimit": 64,
    "qbft": {
      "policy": 0,
      "epoch": 30000,
      "ceil2Nby3Block": 0,
      "testQBFTBlock": 0,
      "blockperiodseconds": 10,
      "emptyblockperiodseconds": 60,
      "requesttimeoutseconds": 20
    }
  },
  "alloc": {
    "0x1872c518405f6b2cf77722c45fa8abdbcffe493a": {
      "balance": "1000000000000000000000000000"
    },
    "0x3bbfb1c24b32fd23d6aea20a9819f9d5031f4d78": {
      "balance": "1000000000000000000000000000"
    },
    "0x75bb662ce8748162bfa79345c74d25d6dd2757ac": {
      "balance": "1000000000000000000000000000"
    },
    "0x7872674fe39002b03dcff6c051bf26c8167eef9b": {
      "balance": "1000000000000000000000000000"
    }
  }
}
```

We are unable to pinpoint what the problem could be.

**Note:** Nothing from the values was changed except the `provider`, `cloudNativeServices` and, the `region` values for the cluster configuration. The `tag` for the Quorum Genesis Tool was also changed from `qgt-0.2.11` to `qgt-0.2.12` since it was not working with the former.

**Steps to Reproduce**

- Clone from the `master` branch
- Create a Kubernetes cluster (we are using AWS for that)
- Install Helm charts as follows:

```
helm install genesis helm/charts/goquorum-genesis --namespace quorum --create-namespace --values helm/values/genesis-goquorum.yml
helm install validator-1 helm/charts/goquorum-node --namespace quorum --values helm/values/validator.yml
helm install validator-2 helm/charts/goquorum-node --namespace quorum --values helm/values/validator.yml
helm install validator-3 helm/charts/goquorum-node --namespace quorum --values helm/values/validator.yml
helm install validator-4 helm/charts/goquorum-node --namespace quorum --values helm/values/validator.yml
```

**Comment by JDawg287** [comment]
One thing I noticed is that the `extraData` field in the `genesis.json` file from one of the nodes does not contain the address from the `address` file.
```
kubectl exec -n quorum goquorum-node-validator-1-0 -- cat ./etc/genesis/genesis.json
```
Output:
```
{
  "nonce": "0x0",
  "timestamp": "0x58ee40ba",
  "extraData": "0xf87aa00000000000000000000000000000000000000000000000000000000000000000f85494a9e4004494058bd02aa41c9123ac8807f5a6171d94324a7c79f2caf5e5b691daf3510edee436eaafae94114daa0681f33a9d830f97a7436ecf5b156a0ac694570795ef5b7466ca4ae244a8c39ecd006479ef18c080c0",
  "gasLimit": "0x47b760",
  "gasUsed": "0x0",
  "number": "0x0",
  "difficulty": "0x1",
  "coinbase": "0x0000000000000000000000000000000000000000",
  "mixHash": "0x63746963616c2062797a616e74696e65206661756c7420746f6c6572616e6365",
  "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
  "config": {
    "chainId": 1337,
    "homesteadBlock": 0,
    "eip150Block": 0,
    "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "eip155Block": 0,
    "eip158Block": 0,
    "byzantiumBlock": 0,
    "constantinopleBlock": 0,
    "petersburgBlock": 0,
    "istanbulBlock": 0,
    "muirglacierblock": 0,
    "berlinBlock": 0,
    "londonBlock": 0,
    "isQuorum": true,
    "maxCodeSizeConfig": [
      {
        "block": 0,
        "size": 64
      }
    ],
    "txnSizeLimit": 64,
    "qbft": {
      "policy": 0,
      "epoch": 30000,
      "ceil2Nby3Block": 0,
      "testQBFTBlock": 0,
      "blockperiodseconds": 10,
      "emptyblockperiodseconds": 60,
      "requesttimeoutseconds": 20
    }
  },
  "alloc": {
    "0x46623a023d1bc2f5bcb3ecc71fbf97c0dc3e31a9": {
      "balance": "1000000000000000000000000000"
    },
    "0xaa1d31aebc1a634c09a2928b407e9d910d9923f1": {
      "balance": "1000000000000000000000000000"
    },
    "0x0d28b72cab775f9ae685fdba8d01753ab0a5a540": {
      "balance": "1000000000000000000000000000"
    },
    "0x95e689439d2af7348329b38d79b8c5ed6885e88c": {
      "balance": "1000000000000000000000000000"
    }
  }
}
```

To check the address in the `address` file:
```
kubectl get cm goquorum-node-validator-1-address -o yaml -n quorum
```
Output:
```
apiVersion: v1
data:
  address: |
    c4a08485cd3e745b69a6ae125c0ecb5c09133f24
kind: ConfigMap
metadata:
  creationTimestamp: "2023-07-05T21:49:28Z"
  name: goquorum-node-validator-1-address
  namespace: quorum
  resourceVersion: "679891"
  uid: 32a49619-ef54-48c0-8311-f8e58c6a0821
  ```
  
As can be seen the address: `c4a08485cd3e745b69a6ae125c0ecb5c09133f24` is not included in the `extraData` field. Is this normal? If not then what is the resolution?

**Note:** I had to restart the pods since I was doing some more testing so the `genesis.json` file is different from the original post.

**Comment by JDawg287** [comment]
**Update:** There were several issues we found with our team. Turns out some of the resources are not being properly deleted after uninstalling and reinstalling the Helm charts. We uninstalled and reinstalled the Helm charts a couple of times.

Not a perfect solution but, we tried to run a network after deleting the resources: `ConfigMaps`, `PersistentVolumeClaims`, `Jobs` etc. Then reinstalled the helm charts and it worked!

If anybody faces the same issue, the best option is to reset the entire namespace and see if it works. Closing the issues with the potential solution.
